libraries/AP_BattMonitor/AP_BattMonitor_FuelLevel_Reventech.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "AP_BattMonitor_Backend.h"
+
+#if AP_BATTERY_FUELLEVEL_REVENTECH_ENABLED
+
+#include <Filter/Filter.h>                     // Filter library
+#include <Filter/AverageFilter.h>      // AverageFilter class (inherits from Filter class)
+#include "AP_BattMonitor.h"
+
+// ADC Calibration for CX31021020 Rev D Voltage Divider for ADC #3
+#define CPN_ADC_5V_SLP 797.76
+#define CPN_ADC_5V_OFF 1.11
+
+// Quadratic fit for LS800 sensor readings to convert to actual fuel in tank
+#define LS800_FIT_FIRST_COEFF 1.0015
+#define LS800_FIT_SECOND_COEFF 0.2419
+#define LS800_FIT_THIRD_COEFF -0.0213
+#define LS800_FIT_OFFSET 0.1766
+
+#define FILTER_MAX 60
+
```

Comment:
Nobody but the .cpp file cares about these; drop them in the implementation file...

Reply:
Done